doc/tutorials/06-active_matter/06-active_matter.tex

```diff
@@ -352,16 +352,16 @@ \subsection{\label{sub:edtut}The Enhanced-Diffusion Tutorial}
 
 pos_id = ParticlePositions(ids=[0])
 msd = Correlator(obs1=pos_id,
-                   corr_operation="square_distance_componentwise",
+                   corr_operation="scalar_product",
```

Comment:
are you sure about this?

Reply:
sorry you are right, we need a subtraction in there

Reply:
I'll leave it as it was then. The correlator output as three components is described in the pdf manual

Reply:
but it's not documented whether the msd is the sum or the average of the three components